tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_header_mode.py

```diff
@@ -0,0 +1,170 @@
+            return 1
+        target_write_pos = 1024
+        target_expire_pos = 2048
+        target_trimmed_pos = 4096
+        # Try to change for write_pos
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank cephfs:0 header set write_pos {target_write_pos}",
+            check_ec=False,
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank cephfs:0 header set expire_pos {target_expire_pos}",
+            check_ec=False,
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank cephfs:0 header set trimmed_pos {target_trimmed_pos}",
+            check_ec=False,
+        )
+        header_out3, ec3 = client1.exec_command(
+            sudo=True,
+            cmd="cephfs-journal-tool --rank cephfs:0 header get",
+            check_ec=False,
+        )
+        header_out3 = json.loads(header_out3)
+        log.info(header_out3)
+        if header_out3["write_pos"] != target_write_pos:
+            log.error("cephfs-journal-tool --rank cephfs:0 header set write_pos failed")
+            return 1
+        if header_out3["expire_pos"] != target_expire_pos:
+            log.error(
+                "cephfs-journal-tool --rank cephfs:0 header set expire_pos failed"
+            )
+            return 1
+        if header_out3["trimmed_pos"] != target_trimmed_pos:
+            log.error(
+                "cephfs-journal-tool --rank cephfs:0 header set trimmed_pos failed"
+            )
+            return 1
+        log.info("change of attributes is successful")
+        log.info("Resetting the values to the original values")
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank cephfs:0 header set write_pos {original_write_pos}",
+            check_ec=False,
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank cephfs:0 header set expire_pos {original_expire_pos}",
+            check_ec=False,
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank cephfs:0 header set trimmed_pos {original_trimmed_pos}",
+            check_ec=False,
+        )
+        header_out4, ec4 = client1.exec_command(
+            sudo=True,
+            cmd="cephfs-journal-tool --rank cephfs:0 header get",
+            check_ec=False,
+        )
+        header_out4 = json.loads(header_out4)
+        log.info(header_out4)
+        if header_out4["write_pos"] != original_write_pos:
+            log.error("cephfs-journal-tool --rank cephfs:0 header set write_pos failed")
+            return 1
+        if header_out4["expire_pos"] != original_expire_pos:
+            log.error(
+                "cephfs-journal-tool --rank cephfs:0 header set expire_pos failed"
+            )
+            return 1
+        if header_out4["trimmed_pos"] != original_trimmed_pos:
```

Comment:
Suggest you to add this code to Reset to defaults under finally section, so that it can get executed even when test fails for some reason before this line.